### DISCUSSION TITLE
ZKLITE DONATION

### DISCUSSION THREAD

**Discussion by Gifty7** [zkSync-Community-Hub/zksync-developers#580] [General]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement and the issue you are running into.

Hey zksync team,
Donated via zklite and holding LIBERTAS OMNIBUS is not included in my allocation, kindly check it, here is my wallet adress 0x938e50bF3826AFEE55a6719D5F160E646df8B7Da 

zklite donation
https://zkscan.io/explorer/transactions/0x1c30d86886cbe4abac64fd37fbf2afa151ca55cf5d2d888e6329c0b9feb9d99a
https://zkscan.io/explorer/transactions/0x1903477308b0bde0ec3b56fd6dac6aa74e6b5a9bb47bf27efe786ac2c3e1a561

**Comment by bxpana** [comment]
Hello,

Thank you for reaching out.

The ZK Token FAQ provides additional details on eligibility criteria: https://docs.zknation.io/zk-token/zk-token-faq

In particular, question 3 ("I met some of the eligibility criteria, but did not get an allocation, why not?") answers your question directly.

**Comment by Gifty7** [comment]
lol
